docs/getting_started/basic/source_files/function.py

```diff
@@ -1,6 +1,8 @@
+"""function for jupyter notebook."""
 from qiskit import QuantumCircuit
-from qiskit.primitives import StatevectorSampler as Sampler
-
+from qiskit.transpiler import generate_preset_pass_manager
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+from qiskit_aer import AerSimulator
```

Comment:
I recommend importing a fake backend from `qiskit_ibm_runtime` instead of instantiating the aer simulator directly. Fake backends contain useful information for the transpiler and the workflow is more similar to what a workflow with a real backend would look like. For example, you can pick `FakeVigoV2` as a 5 qubit fake backend:

```suggestion
from qiskit_ibm_runtime.fake_provider import FakeVigoV2
```